src/docker.js

```diff
@@ -105,6 +106,9 @@ docker.buildImage = function(project, tarPath, imageId, buildId, buildLogger, do
         }
 
         buildLogger.info('[%s] %s', tag, result.stream || result.status);
+        if (result.stream && result.stream.indexOf('Successfully built ') == 0) {
+          realBuildId = result.stream.split('Successfully built ')[1].replace('\n', '');
```

Comment:
this is very hack-ish, it's prone to error :) maybe check if dockerode provides other ways to give you the image name?